allennlp/data/dataset_readers/dataset_reader.py

```diff
@@ -38,20 +38,20 @@ class WorkerInfo:
 @dataclass
 class DistributedInfo:
     """
-    Contains information about the node rank and world size when the reader is being
+    Contains information about the process rank and world size when the reader is being
```

Comment:
You call it "process rank" here, but `global_rank` in the code.  Would `process_rank` be better?

Reply:
I think having the word "global" is important to be clear what the rank represents. I'll update the terminology in the doc comment.